### ISSUE TITLE
A0Auth0 object undefined

### ISSUE THREAD

**Issue by Abdisalan** [auth0/react-native-auth0#63]
I installed react-native and this library but when I tried running I got:

`[TypeError: undefined is not an object (evaluating 'A0Auth0.oauthParameters')]`

I tried looking into this and it seems that in `webauth/index.js` on line 11 the react-native object `NativeModules` does not contain `A0Auth0`. I looked into the react-native repo code and found no mention of `A0Auth0` so maybe this repo isn't compatible with the latest version of react-native?

What's the fix for this?

**Comment by hzalaz** [comment]
@Abdisalan did you follow the install instructions detailed in the README?

**Comment by Abdisalan** [comment]
Yes, I added the changes to my AppDelegate.m and added the xml piece to my Info.plist file.

**Comment by hzalaz** [comment]
@Abdisalan and `react-native link react-native-auth0`?

**Comment by Abdisalan** [comment]
Oh I forgot that step! That fixed it. Thank you

**Comment by igorlamos** [comment]
For those of you experiencing the same problem but you are sure you run `react-native link react-native-auth0`, try:
- `react-native unlink react-native-auth0`
- `react-native link react-native-auth0`

It will unlink the project first, then link it again ... it's the way I solved this for Android. I had used  `react-native-lock` previously and there was "some problem" when first running **link** for `react-native-auth0`.

Hope it helps someone.
